systemd-udevd/pkg.yaml

```diff
@@ -0,0 +1,114 @@
+        patch -p1 < /pkg/patches/0008-Use-uintmax_t-for-handling-rlim_t.patch
+        patch -p1 < /pkg/patches/0009-don-t-pass-AT_SYMLINK_NOFOLLOW-flag-to-faccessat.patch
+        patch -p1 < /pkg/patches/0010-Define-glibc-compatible-basename-for-non-glibc-syste.patch
+        patch -p1 < /pkg/patches/0011-Do-not-disable-buffering-when-writing-to-oom_score_a.patch
+        patch -p1 < /pkg/patches/0012-distinguish-XSI-compliant-strerror_r-from-GNU-specif.patch
+        patch -p1 < /pkg/patches/0013-avoid-redefinition-of-prctl_mm_map-structure.patch
+        patch -p1 < /pkg/patches/0014-do-not-disable-buffer-in-writing-files.patch
+        patch -p1 < /pkg/patches/0016-Handle-missing-gshadow.patch
+        patch -p1 < /pkg/patches/0017-missing_syscall.h-Define-MIPS-ABI-defines-for-musl.patch
+        patch -p1 < /pkg/patches/0019-Adjust-for-musl-headers.patch
+        patch -p1 < /pkg/patches/0022-sd-event-Make-malloc_trim-conditional-on-glibc.patch
+        patch -p1 < /pkg/patches/0023-shared-Do-not-use-malloc_info-on-musl.patch
+        patch -p1 < /pkg/patches/0024-avoid-missing-LOCK_EX-declaration.patch
+        patch -p1 < /pkg/patches/0025-include-signal.h-to-avoid-the-undeclared-error.patch
+        patch -p1 < /pkg/patches/0026-undef-stdin-for-references-using-stdin-as-a-struct-m.patch
+        patch -p1 < /pkg/patches/0027-adjust-header-inclusion-order-to-avoid-redeclaration.patch
+        patch -p1 < /pkg/patches/0028-build-path.c-avoid-boot-time-segfault-for-musl.patch
+
+        # Own patches
+        patch -p1 < /pkg/patches/0001-chore-musl-headers-compatibility-for-udev.patch
+
+        # for full build add this:
+        #-Dnss-mymachines=disabled -Dnss-myhostname=false -Dnss-resolve=disabled -Dnss-systemd=false
+        meson setup build \
+          --buildtype=release \
+          -Dmode=release \
+          -Dversion-tag="{{ .systemd_version }}-talos" \
+          -Dselinux=enabled \
+          -Dblkid=enabled \
+          -Dkmod=enabled \
+          -Dopenssl=disabled \
+          -Dgshadow=false \
+          -Dutmp=false \
+          -Duserdb=false \
+          -Ddbus=disabled \
+          -Dglib=disabled \
+          -Dlink-udev-shared=false \
+          -Dman=disabled \
+          -Dtests=false \
+          -Dwerror=false
+    build:
+      - |
+        # rules.d/64-btrfs.rules is disabled in our build, other are static
+        # Buildable rules from rules.d/meson.build
+        ninja -j $(nproc) -C build udevadm rules.d/50-udev-default.rules rules.d/71-seat.rules rules.d/90-vconsole.rules \
+          rules.d/60-persistent-storage.rules rules.d/70-uaccess.rules rules.d/73-seat-late.rules rules.d/99-systemd.rules \
+          ata_id cdrom_id dmi_memory_id fido_id iocost mtd_probe scsi_id v4l_id \
+          libudev.so.1.7.9 libudev.a src/libudev/libudev.pc src/udev/udev.pc
+        # Build all instead and copy necessary parts?
+        # ninja -j $(nproc) -C build
```

Comment:
Here we either have to build 876 targets with a bulky command we need to update on some systemd updates (.so version change, new/removed udev rules) or build 1318 targets together, do ninja install and then copy directories we need into the image

Reply:
If that doesn't require more patches, I'd say build all and copy what we need (will be easier to maintain in the future).

Do systemd udev rules depend on something like `systemd-run`?

Reply:
> Do systemd udev rules depend on something like systemd-run?

Looks like no, but I might need to proofread them anyway as we got a few more we are likely to not need (seat stuff, 99-systemd)

Reply:
Ok, here's the file tree of meson install results. https://tpaste.us/DrEo

Seems easier to maintain, and I'll try to minimize the components installed even more